docs/tutorial.txt

```diff
@@ -15,4 +15,4 @@ Tutorials
    /tutorial/indexes
    /tutorial/tailable-cursor
    /tutorial/example-data
-   /tutorial/versioned-api
+   /tutorial/stable-api
```

Comment:
@mongodb/docs-platform Is there an easy way to implement a web redirect from `versioned-api` to `stable-api`? Or is this something the docs team would handle themselves in https://github.com/mongodb/docs-php-library?

I'm not terribly concerned about this, as I doubt we have many direct links to this tutorial out there.

Reply:
It would normally be handled in the docs-php-library repo  in the config/redirects file!

Reply:
@schmalliso: Thanks, I'll make a note of this after merging and opening a DOCSP ticket requesting a deployment.